RecoCTPPS/PixelLocal/python/ctppsPixelLocalReconstruction_cff.py

```diff
@@ -1,15 +1,21 @@
 import FWCore.ParameterSet.Config as cms
 
 # geometry
-#from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+#from Geometry.VeryForwardGeometry.geometry_CTPPS_alaTotem_RECO_cfi import *
```

Comment:
please remove this commented import.
It looks like geometry_CTPPS_alaTotem_RECO_cfi is not even in the release.